tests/integration/bond_test.py

```diff
@@ -99,6 +100,16 @@ def bond99_with_slave(eth2_up):
         yield state
 
 
+def test_remove_existing_bond(setup_remove_bond99):
+    subprocess.check_call(["ip", "link", "add", BOND99, "type", "bond"])
```

Comment:
Could you please use `from .testlib import cmd as libcmd` instead?
This also fits a fixture as it is not part of the test itself (it is a setup of the test).

Reply:
What is the benefit of using `cmd` here?